src/main/java/software/amazon/awssdk/s3accessgrants/cache/S3AccessGrantsConstants.java

```diff
@@ -28,11 +28,9 @@ public class S3AccessGrantsConstants {
     public static final int ACCESS_DENIED_CACHE_SIZE = 3_000;
 
     public static final int BUCKET_REGION_CACHE_SIZE = 1_000;
-
     public static final int MAX_BUCKET_REGION_CACHE_SIZE = 1_000_000;
+    public static final int BUCKET_REGION_EXPIRE_CACHE_AFTER_WRITE_SECONDS= 3_600; // 1 hour
```

Comment:
Why are we increasing the bucket cache durations? I remember receiving feedback to not to do this based on SDK teams previous experiences.